includes/admin/class-wc-admin-addons.php

```diff
@@ -71,9 +85,19 @@ public static function build_parameter_string( $category, $term, $country ) {
 	 */
 	public static function get_extension_data( $category, $term, $country ) {
 		$parameters     = self::build_parameter_string( $category, $term, $country );
+
+		$headers = array();
+		$auth = WC_Helper_Options::get( 'auth' );
+
+		if ( ! empty( $auth['access_token'] ) ) {
+			$headers['Authorization'] = 'Bearer ' . $auth['access_token'];
+		}
+
 		$raw_extensions = wp_remote_get(
```

Comment:
@raicem could we change this to `wp_safe_remote_get`?

Reply:
I agree that is a good idea.

Reply:
Ah, sorry. I have overlooked this. Fixed with this commit: https://github.com/woocommerce/woocommerce/pull/28719/commits/eb3ba820dab8e3c663e26d30e11d2e27714ad6fb